### DISCUSSION TITLE
README

### DISCUSSION THREAD

**Discussion by giscus** [xdd1997/docsify#15] [General]
# README

https://www.20161101.xyz/docsify/#/

**Comment by mzlogin** [comment]
评论测试

**Reply by xdd1997** [reply]
评论后的内容可以出现在 https://github.com/xdd1997/docsify/discussions 但是不能显示在评论的网页上，你知道为啥吗

**Reply by mzlogin** [reply]
在另一处回复你了：https://github.com/mzlogin/blog-comments/discussions/204?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDQ2ODc0ODczMDQ6MTY0NjU5MA%3D%3D#discussioncomment-4837143
